markdown/developers/guidelines.md

```diff
@@ -8,12 +8,22 @@ subtitle: Guidelines and requirements for nf-core pipelines.
 If you're thinking of adding a new pipeline to nf-core, please read the documentation
 about [adding a new pipeline](/developers/adding_pipelines).
 
-## Workflow size
+## Workflow size and specificity
 
 We aim to have a _"not too big, not too small"_ rule with nf-core pipelines.
-This is a little fuzzy, but as a rule of thumb workflows should contain at
+This is deliberately fuzzy, but as a rule of thumb workflows should contain at
 least three different processes and be simple enough to run that a new user
 can realistically run the pipeline after spending ten minutes reading the docs.
+Pipelines should be general enough to be of use to multiple groups and research
+projects, but comprehensive enough to cover most steps in a typical analysis.
```

Comment:
Have we skipped the "primary analysis" part? "Most steps in a typical analysis" might be pushing it a bit too far?

Reply:
Yeah, "primary" is a better description here.. Will update 👍